src/sentry/models/artifactbundle.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_only_model,
+)
+from sentry.models import DB_VERSION_LENGTH
+
+
+@region_silo_only_model
+class ArtifactBundle(Model):
+    __include_in_export__ = False
+
+    bundle_id = models.UUIDField(null=True)
+    organization_id = BoundedBigIntegerField(db_index=True)
+    release_name = models.CharField(max_length=DB_VERSION_LENGTH, null=True)
+    dist_id = BoundedBigIntegerField(null=True)
+    file = FlexibleForeignKey("sentry.File")
+    artifact_count = BoundedPositiveIntegerField()
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_artifactbundlefile"
+
+        unique_together = (
+            ("organization_id", "bundle_id"),
+            ("organization_id", "release_name"),
+        )
+        index_together = (("organization_id", "release_name"),)
```

Comment:
You can remove this index since this set of columns is already in `unique_together`, which creates an index implicitly to enforce the uniqueness.